docs/validators/emergency-upgrades.md

```diff
@@ -0,0 +1,86 @@
+
+If there are any serious issues with the upgrade then you will notified on Discord to rollback the changes.
+
+#### **Rolling back an upgrade**
+
+During the network upgrade, the relevant Cosmos teams will be monitoring the [cosmos-hub-validators-verified](https://discord.com/channels/669268347736686612/798937713474142229)  channel and communicating with validators on the status of the upgrade.
+
+During this time, the relevant teams will listen to validator needs to determine if the upgrade is experiencing unintended challenges. In the event of unexpected challenges, the relevant teams, after conferring with validators, may choose to declare that the upgrade will be skipped.
+
+**Note**: There is no particular need to restore a state snapshot prior to the upgrade height, unless specifically directed by relevant teams.
+
+### **Vulnerability Levels**
+
+This section discusses the actions to be taken for different vulnerability levels.
+
+#### **Critical level vulnerabilities**
+
+For critical level vulnerabilities, the relevant teams will usually start a private patching effort with a number of key validators. By the time an emergency patch is released, the expectation is that a good proportion of the validator set have already upgraded. Vulnerabilities at this level should be updated immediately. We might ask validators to shut down the network until a patch is released, due to the potential consquences arising from the identified vulnerability.
+
+#### **High level vulnerabilities**
+
+For high level vulnerabilities, the relevant teams will reach out to validators to upgrade the chain asap.
+
+#### **Medium level vulnerabilities**
+
+For medium level vulnerabilities, the team will release a patch and may consider an out-of-band (emergency) release. We encourage validators to upgrade asap.
+
+#### **Low level vulnerabilities**
+
+Fixes for low level vulnerabilities are periodically rolled into point releases or if state breaking, they are rolled into the next minor or major release.
+
+## **What to expect before the upgrade?**
+
+Once a patch is ready, the relevant teams will reach out to all the validators via a number of pre-established public or private channels depending on the severity of the vulnerability.
+
+Validators **MUST HAVE** their contact details on file by registering their teams on [Cosmos Developers Discord](https://discord.gg/cosmosnetwork) and join the [cosmos-hub-validators-verified](https://discord.com/channels/669268347736686612/798937713474142229) channel. This is necessary so that validators don't miss important updates and get jailed for missing blocks or have other operational issues.
+
+## **Main Contact Points**
+
+The primary contact points for the Cosmos Hub are the relevant teams at Informal Systems and the [Cosmos Developers Discord](https://discord.gg/cosmosnetwork) on the [cosmos-hub-validators-verified](https://discord.com/channels/669268347736686612/798937713474142229) channel.
```

Comment:
How might one reach the relevant teams at Informal Systems? 

Is there an email maintained specifically for security comms cc: @greg-szabo

Reply:
suggest contacts at:

* informal
* icf
* notional
* amulet

reasoning: one org could be hit by a bus, etc, and this way, we don't have that problem.